UT-Alignment-12d-1/readme.md

```diff
@@ -0,0 +1,38 @@
+
+### Intent
+
+IFC 4x3 RC2
+This scenario has a simple alignment with left arc in horizontal and no vertical.
+
+That is it has Civil Horizontal Geometry (HG):
+
+- straight
+- an arc with ifc positive radius of curvature
+- a straight
+
+In the ifc file there is only the ifc semantic definition of the alignment and no ifc geometry.
+The Ifc file was generated by 12d Model. 
```

Comment:
You're using all three possibilities as to how to spell the word "IFC":

- ifc
- Ifc
- IFC

within one document. I'd prefer always using IFC for consistency.

I agree that the IFC entities follow CamelCase notation.